src/common/snippets/src/lowered/pass/softmax_decomposition.cpp

```diff
@@ -58,15 +62,24 @@ bool SoftmaxDecomposition::run(LinearIR& linear_ir) {
             // Init value of vector buffer for ReduceMax is -FLOAT_MIN.
             const auto fill_max = push_node(std::make_shared<op::Fill>(vector_buffer_max.second, 0, float_min_constant));
             // ReduceMax loop
-            const auto& max = push_node(std::make_shared<ov::op::v1::Maximum>(softmax->get_input_source_output(0), fill_max.second));
+            const auto fill_max_tail = push_node(std::make_shared<op::Fill>(softmax->get_input_source_output(0), inner_increment, float_min_constant));
```

Comment:
Probably I missed something in the PR and couldn't find the answer, but... Could you tell me please if this `Fill` is left on the main loop too? If it's so, I think we have to update the emitter implementation: add 'uni_vmovups(...);' [here](https://github.com/openvinotoolkit/openvino/blob/02c47e47b22ffaf6d7fde518993e04cbcfc418a6/src/plugins/intel_cpu/src/emitters/snippets/x64/jit_fill_emitter.cpp#L62).

Reply:
You are right, thank you! The emitter implementation is updated according with the suggestion. Could you please double check?